README.md

```diff
@@ -56,6 +56,36 @@ Using this method, you can `require` twilio-video.js like so:
 const Video = require('twilio-video');
 ```
 
+TypeScript definitions can now be imported using this method.
+
+```ts
+import * as Video from 'twilio-video';
+
+Video.createLocalTracks().then((localTracks: Array<Video.LocalAudioTrack|Video.LocalVideoTrack>) => {
```

Comment:
I'm still on the fence about including an example to show users how to import types. 

The main reason is - most of the time, users won't need to import our type definitions in order to use them. 

For instance, in this code example, we don't actually need to provide type annotations ourselves because TS handles this for us. The following example is valid typescript, and the IDE is fully aware of the types of all variables used.

```
Video.createLocalTracks().then(localTracks => {
  return Video.connect('token', {
    name: 'my-cool-room',
    tracks: localTracks
  });
}).then(room => {
  console.log('Successfully connect to Room:', room.name);
});
```

I wonder if we should use an example where we are actually required to annotate the types. For instance, we could use this code from the 'Usage' section of the Readme:

```
import { RemoteParticiant } from 'twilio-video';

function participantDisconnected(participant: RemoteParticipant) {
  console.log('Participant "%s" disconnected', participant.identity);
  document.getElementById(participant.sid).remove();
}
```

Here, the `participantDisconnected` function doesn't actually know what the `participant` argument is supposed to be, so we need to annotate it ourself. 

I also don't think that we need to explain the difference bewteen `import { RemoteParticiant } from 'twilio-video';` and `import * as Video from 'twilio-video'`. This has to do with import syntax, which is just plain JavaScript and (in my opinion) it doesn't really relate to Type annotations or Twilio Video. I think we can assume that developers know how to import things. The main idea here is that we need to inform users that types are now importable.